styles/Styles.css

```diff
@@ -1,55 +1,176 @@
-@import url(http://localhost:63342/frontend-html-css-concept-opdracht/index.html?_ijt=k5p124bovrkespsqmd2a4o23m2&_ij_reload=RELOAD_ON_SAVE);
 @import url('https://fonts.googleapis.com/css2?family=Merriweather&display=swap');
-*{
+* {
     box-sizing: border-box;
     padding: 0;
     margin: 0;
 }
-header{
+header {
     font-family: Merriweather, sans-serif;
     background-color: #4845E4;
     color: aliceblue;
     text-align: center;
     height: 500px;
 }
-.nav-1{
+.nav-1 {
     display: flex;
     flex-direction: row;
     flex-basis: 200px;
     justify-content: flex-end;
 
 }
-ul{
+ul {
 display: flex;
     list-style: none;
     justify-content: space-around;
     flex-basis: 400px;
 }
-h1{
+h1 {
     margin-top: 80px;
 }
-h4{
-    margin: 30px;
-}
-.button-1{
+
+.button-1 {
     background-color: #ED4385;
     border-radius: 50px;
     font-family: inherit;
     color: aliceblue;
     padding: 20px 40px;
     border-style: none;
     box-shadow: 2px 2px 2px 2px rgba(17,15,17,0.27);
+    margin-bottom: 60px;
+}
+a {
+    text-decoration: none;
+    color: #5722CD;
+    font-size: 12px;
+    font-weight: bold;
 }
-main{
```

Comment:
Wat ik heb getracht is om zoveel mogelijk margin te vermijden. 

Door gebruik te maken van de flexbox zou je ook gap kunnen gebruiken om de ruimtes te verdelen.

Bij mij ook niet overal gelukt maar zou je nog kunnen overwegen.